x/programs/rust/examples/token/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-expose_macro = { version = "0.1.0", path = "../../expose_macro" }
+public_macro = { version = "0.1.0", path = "../../public_macro" }
```

Comment:
You should also expose this from the SDK instead of making all the examples depend on it directly